src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsmanageadjudicationsapi/services/reported/NomisHearingOutcomeService.kt

```diff
@@ -0,0 +1,33 @@
+package uk.gov.justice.digital.hmpps.hmppsmanageadjudicationsapi.services.reported
+
+import jakarta.transaction.Transactional
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.hmppsmanageadjudicationsapi.entities.HearingOutcome
+import uk.gov.justice.digital.hmpps.hmppsmanageadjudicationsapi.entities.HearingOutcomeCode
+import uk.gov.justice.digital.hmpps.hmppsmanageadjudicationsapi.gateways.PrisonApiGateway
+import uk.gov.justice.digital.hmpps.hmppsmanageadjudicationsapi.repositories.HearingRepository
+import uk.gov.justice.digital.hmpps.hmppsmanageadjudicationsapi.repositories.ReportedAdjudicationRepository
+
+@Transactional
+@Service
+class NomisHearingOutcomeService(
+  private val prisonApiGateway: PrisonApiGateway,
+  private val reportedAdjudicationRepository: ReportedAdjudicationRepository,
+  private val hearingRepository: HearingRepository,
+) {
+
+  fun checkForNomisHearingOutcomesAndUpdate() {
+    hearingRepository.findByHearingOutcomeIsNull().forEach { hearing ->
+      if (prisonApiGateway.doesHearingOutcomeExist(
```

Comment:
should this be `hearingOutcomeExistsInNomis() : Boolean` function?

Reply:
thanks.  yeah lets rename, was just putting it in for now, but yeah, needs an endpoint that the cron can call etc

This function is simply to run the call to nomis, and update if needs be (so the gateway facade is the Boolean one).  Can resolve once we have prison api and integrate